### PR TITLE
Add deploy clean target and rename es/cass to deploy-

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

**Comment by jpkrohling** [comment]
This change is [Reviewable](https://reviewable.io/reviews/jaegertracing/jaeger-operator/149)

File: Makefile [hunk 1]
```diff
@@ -81,20 +81,26 @@ run: crd
 run-openshift: crd
 	@bash -c 'trap "exit 0" INT; OPERATOR_NAME=${OPERATOR_NAME} KUBERNETES_CONFIG=${KUBERNETES_CONFIG} WATCH_NAMESPACE=${WATCH_NAMESPACE} go run -ldflags ${LD_FLAGS} main.go start --platform=openshift'
 
-.PHONY: es
-es:
-	@kubectl create -f ./test/elasticsearch.yml 2>&1 | grep -v "already exists" || true
```
Comment (pavolloffay) [inline Makefile]: This was not grepping anything.

**Comment by pavolloffay** [comment]
I have reverted back grep part

**Comment by pavolloffay** [comment]
> . If we see people being confused about what the es/cassandra/ingress targets do, we can then consider adding a deploy- prefix. Otherwise, let's keep the simpler version.

I was the first one confused by these targets. I thought you compile Cassandra here!@ :fearful:

**Comment by pavolloffay** [comment]
updated to your preference

**Pull request merged**